### ISSUE TITLE
Google Gemini not working after the latest update #195

### ISSUE THREAD

**Issue by ysid27** [microsoft/TaskWeaver#203]
**Describe the bug**
Google Gemini is no longer working after the latest [update](https://github.com/microsoft/TaskWeaver/commit/6e0e3e6bdb828152ddcc2241935bf930868a21cd) #195 

**To Reproduce**
- Follow these [instructions](https://microsoft.github.io/TaskWeaver/docs/llms/gemini/) to setup Google Gemini in TaskWeaver.
- Follow these [instructions](https://microsoft.github.io/TaskWeaver/docs/usage/cmd) to run TaskWeaver from CMD.

**Error**
`Error: Cannot process your request due to Exception: Please let the response complete iteration before accessing the final accumulated`

***More Details*
```
raise IncompleteIterationError(_INCOMPLETE_ITERATION_MESSAGE)
google.generativeai.types.generation_types.IncompleteIterationError: Please let the response complete iteration before accessing the final accumulated
attributes (or call `response.resolve()`)
```

**Additional context**
I used the latest TaskWeaver library including the latest google_genai.py file. It failed with the above error. 
Then I tried using the old google_genai.py with the latest TaskWeaver library and it worked flawlessly. So I think the update is causing the issue.

**Comment by bpawnzZ** [comment]
I confirm this is still an issue

**Comment by zhangxu0307** [comment]
Just fixed this bug in #206
